Automattic/sensei
# Enable translating more Sensei content using Polylang

### Changes proposed in this Pull Request

* Enables translating modules, questions and question categories.
* Enables translating user-defined values specified in settings.

### Testing instructions

* Install [Polylang](https://en-ca.wordpress.org/plugins/polylang/) and go through the setup wizard. Ensure you select at least 2 languages.
* Ensure modules, questions and question categories can all be translated.
* Go to _Languages_ > _Strings translations_.
* Filter by `sensei-lms` and ensure there are entries for `course_archive_more_link_text` and `email_footer_text`.

~Note that our `wpml-config.php` file is not actually used by WPML. WPML has a [hosted version](https://github.com/OnTheGoSystems/wpml-config/blob/master/sensei/wpml-config.xml) they use instead, so this PR, ironically enough, doesn't affect WPML.~

Turns out the hosted version is overridden by our local version, so this affects WPML too.
